src/components/questions/MultipleChoiceList.tsx

```diff
@@ -1,66 +1,158 @@
-import React from 'react';
-import { StyleSheet } from 'react-native';
+import React, { useState } from 'react';
+import { Animated, StyleSheet, Text, TouchableOpacity, View } from 'react-native';
 import Icon from 'react-native-vector-icons/FontAwesome';
-import RadioButtonRN from 'radio-buttons-react-native';
 import { QuestionOption } from '../../models/QuestionOption';
 import FONTS from '../../assets/fonts';
 import COLORS from '../../assets/colors';
 import { MultipleChoiceExtraData } from '../../models/questionOptionExtraData/MultipleChoiceExtraData';
 
-interface RadioButtonData {
-  label: string;
-}
-
 const MasterContainer = (props: {
   values?: string[];
   questionOption: QuestionOption;
-  onClicked: (label: string) => void;
+  onClicked: (values: string[]) => void;
 }) => {
-  const multipleChoiceQuestions = (props.questionOption.extra_data as MultipleChoiceExtraData)
-    .values;
+  const multipleChoiceOptions: string[] = (
+    props.questionOption.extra_data as MultipleChoiceExtraData
+  ).values;
+  const multipleAnswersPossible: boolean = (
+    props.questionOption.extra_data as MultipleChoiceExtraData
+  ).multiple;
+  const [currentValues, setCurrentValues] = useState(props.values ?? []);
+  let fadeAnim = new Animated.Value(0);
+
+  const selectOption = (value: any, index: number) => {
+    fadeAnim.setValue(index);
+    if (currentValues.includes(value)) {
+      value = currentValues.filter((currentValue) => {
+        return currentValue !== value;
+      });
+      setCurrentValues(value);
+      // fadeOutAnimation();
+    } else if (multipleAnswersPossible || currentValues.length === 0) {
+      value = [...currentValues, value];
+      setCurrentValues(value);
+      // fadeInAnimation();
+    }
+    props.onClicked(value);
+  };
+
+  // const fadeInAnimation = () => {
```

Comment:
Kan weg denk ik?

Reply:
Ligt eraan of we de animatie nog willen toevoegen.

Reply:
Zag er zonder animatie prima uit. De rest van de app heeft toch geen animatie, dus maakt vrij weinig uit.